backend/api/auth.py

```
@@ -1,167 +1,441 @@
+            raise ValueError("Access key cannot be empty")
+            
+        # AWS access keys follow specific patterns - validate with comprehensive regex
+        import re
+        
+        # AWS access key pattern: Known AWS prefixes + 16 alphanumeric characters
+        # AKIA: Standard access keys, ASIA: STS temporary, AROA: Role, AIDA: IAM user, AIPA: Service roles
+        aws_key_pattern = r'^(AKIA|ASIA|AROA|AIDA|AIPA)[A-Z0-9]{16}$'
+        
+        cleaned_key = v.replace(' ', '')
+        if not re.match(aws_key_pattern, cleaned_key):
+            raise ValueError("Invalid AWS access key format")
```

Ruff (0.14.14)

Avoid specifying long messages outside the exception class

(TRY003)